js/libs/form.js

```diff
@@ -291,7 +294,11 @@ Fliplet().then(function () {
           }
           this.isSent = false;
         },
-        reset: function(trackEvents) {
+        reset: function () {
+            isResetAction = true;
```

Comment:
Hi @tonytlwu,

We are setting isResetAction flag initially to false, and on click of clear button we are setting isResetAction flag to true. 

If form has both image and file component or mulitple images/files, in this case we dont want to reset flag to false because[ switch case](https://github.com/Fliplet/fliplet-widget-form-builder/blob/master/js/libs/form.js#L195-L206) will not execute for next image / file fields.

Reply:
@NikitaJadhavXoriant This looks like if `reset()` is run, `isResetAction` is changed from `false` to `true`. The next time `submit()` is run (which calls `resetForm()`, the `isResetAction` variable will still be `true` -- I don't think that's what we want, right?

Reply:
@tonytlwu Agree, missed this scenario. Adding isResetAction to false in onSubmit method.